test/e2e/upgrade/upgrade.go

```diff
@@ -136,10 +137,19 @@ func SetUpgradeAbortAt(policy string) error {
 	return fmt.Errorf("abort-at must be empty, set to 'random', or an integer in [0,100], inclusive")
 }
 
-var _ = g.Describe("[sig-arch][Feature:ClusterUpgrade]", func() {
+var _ = g.Describe("[sig-arch][Feature:ClusterUpgrade]", g.Ordered, func() {
+	var upgradeErr error
 	f := framework.NewDefaultFramework("cluster-upgrade")
 	f.SkipNamespaceCreation = true
 
+	g.It("Cluster should be upgradeable before beginning upgrade", func() {
```

Comment:
As I recall, there is a CVO test that makes sure that a cluster that is not upgradeable can be forced.  Unfortunately, Jack retired, but I think @wking reviewed it.